data-processing/scripts/process_pdf.py

```diff
@@ -1,3 +1,6 @@
+# TODO: CONSIDER REMOVING ENTIRE FILE, doesn't look like it's even being used anymore.
+#  the import upload_citations from entities.citations.util has been deleted in July 2020.
```

Comment:
THIS IS THE ONLY CHANGE I made in this file.

Reply:
Looks like it maybe also got renamed?

Reply:
that's what i thought at first, but the existing `load_located_citations()` in `entities.citations.util.py` has nothing to do with this.  Looks like there was a whole bunch of refactoring going on for that [commit](https://github.com/allenai/scholarphi/commit/a4f8285d6f443f2c3b97dc12f9098794c89d79f3) over 17 files anyway.